<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>610</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>461</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>MOBILE DETECTION CAMERA</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>FILE</string>
    </property>
    <widget class="QLineEdit" name="FilePathURI">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="browseImageFileButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse File</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>421</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>CAMERA</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CAMERA RTSP URI</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CAMERA TYPE</string>
     </property>
    </widget>
    <widget class="QComboBox" name="predefinedMOBDropDown">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>HIKVISION</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DAHUA</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="MOBcameraRTSPURL">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="ImageModeDropdown">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>CAMERA RTSP URL</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MODE</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>461</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>PEDESTRIAN CAMERA</string>
   </property>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>FILE</string>
    </property>
    <widget class="QLineEdit" name="FilePathURI_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="browseImageFileButton_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse File</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>421</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>CAMERA</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CAMERA RTSP URI</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CAMERA TYPE</string>
     </property>
    </widget>
    <widget class="QComboBox" name="predefinedPEDDropDown">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>HIKVISION</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DAHUA</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="PEDcameraRTSPURL">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="ImageModeDropdown_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>CAMERA RTSP URL</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MODE</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>buttonBox</zorder>
  <zorder>groupBox_4</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
